api/tests/test_full_openapis.py

```diff
@@ -10,11 +10,10 @@
 
 
 @pytest.mark.parametrize(
-    "spec_file_path",
-    list_files_in_api_specs("*.json") + list_files_in_api_specs("*.y*ml"),
+    "spec_file", list_files_in_api_specs("*.json") + list_files_in_api_specs("*.y*ml")
 )
-def test_valid_openapi_specs(spec_file_path):
-    spec_file_path = Path(spec_file_path)
+def test_valid_openapi_specs(spec_file: str):

```

Comment:
what about using pathlib instead of str?

Reply:
it can be done. changed it